src/Datadog.Trace/AsyncLocalScopeManager.cs

```diff
@@ -49,17 +49,18 @@ public void Close(Scope scope)
             if (current == null || current != scope)
             {
                 // This is not the current scope for this context, bail out
+                SpanClosed?.Invoke(this, new SpanEventArgs(scope.Span));
                 return;
             }
 
             // if the scope that was just closed was the active scope,
             // set its parent as the new active scope
-            _activeScope.Set(current.Parent);
-            SpanDeactivated?.Invoke(this, new SpanEventArgs(current.Span));
+            _activeScope.Set(scope.Parent);
```

Comment:
Why the change to scope here?

Reply:
At this point we know that scope==current, and I wanted to have consistency in the variable name we used to make it easier to read. Everywhere we create an SpanEventArgs we use scope now so it seemed like a nice refactor.